.github/workflows/publish.yml

```diff
@@ -37,7 +37,6 @@ jobs:
   deploy-prod:
```

Comment:
this step is no longer doing the actual deployment of the stac, could it be renamed and description updated. 

Is this also considered two prod deploys as there are two environment: prod ?

Reply:
> Is this also considered two prod deploys as there are two environment: prod ?

What actually counts prod deploys? Is it a problem if there are multiple deploys?

Reply:
was mostly wondering what would show up in the deployments table https://github.com/linz/imagery/deployments/prod having two records in there per commit isn't a problem, just makes it slightly harder to follow

Reply:
I don't know. I also don't know whether the `sync-stac` job will even run if it doesn't have the same `environment` as the dependee job.